api/lib/infrastructure/repositories/membership-repository.js

```diff
@@ -39,92 +25,91 @@ function _setSearchFiltersForQueryBuilder(filter, qb) {
   }
 }
 
+async function _buildMembership(membershipAttributes) {
```

Comment:
_toDomain ça te plait pas ? juste pour une histoire d'homogénéité

Reply:
ça utilise pas le bookshelf-to-domain-converter.js je trouve que to domain c'est soouvent lié à ça, pas chaud pour utiliser le même nom, et buildMembership c'est clair sur l'intention de la fonction.